packages/react-components/src/templates/ResearchOutputPage.tsx

```diff
@@ -1,30 +1,124 @@
 import React from 'react';
 import css from '@emotion/css';
 import { ResearchOutputResponse } from '@asap-hub/model';
+import format from 'date-fns/format';
 
-import { Display, TagLabel } from '../atoms';
+import { TagLabel, Link, Paragraph, Card, Caption, Headline3 } from '../atoms';
 import { RichText } from '../organisms';
-import { pearl } from '../colors';
-import { perRem } from '../pixels';
+import { lead } from '../colors';
+import {
+  perRem,
+  mobileScreen,
+  largeDesktopScreen,
+  vminLinearCalc,
+} from '../pixels';
 import { contentSidePaddingWithNavigation } from '../layout';
+import { teamIcon, chevronCircleLeftIcon } from '../icons';
+
+const teamMemberStyles = css({
+  color: lead.rgb,
+  display: 'flex',
+  alignItems: 'center',
+  paddingTop: `${12 / perRem}em`,
+  paddingBottom: `${24 / perRem}em`,
+});
+
+const iconStyles = css({
+  display: 'inline-block',
+  width: `${24 / perRem}em`,
+  height: `${24 / perRem}em`,
+  paddingRight: `${15 / perRem}em`,
+});
 
 const containerStyles = css({
-  alignSelf: 'stretch',
-  background: pearl.rgb,
   padding: `${36 / perRem}em ${contentSidePaddingWithNavigation(8)}`,
 });
 
-const ResearchOutputPage: React.FC<ResearchOutputResponse> = ({
+const backContainerStyles = css({
+  alignSelf: 'flex-start',
+  padding: `${30 / perRem}em 0 `,
+});
+
+const backButtonStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const cardPaddingStyles = css({
+  padding: `${24 / perRem}em ${vminLinearCalc(
+    mobileScreen,
+    24,
+    largeDesktopScreen,
+    36,
+    'px',
+  )}`,
+});
+
+const visibilityStyles = css({
+  paddingTop: `${24 / perRem}em`,
+  paddingBottom: `${30 / perRem}em`,
+});
+
+const postedStyles = css({
+  color: lead.rgb,
+});
+
+type ResearchOutputPageProps = Pick<
+  ResearchOutputResponse,
+  'publishDate' | 'title' | 'type' | 'created' | 'team' | 'text'
+> & {
+  team?: {
+    href: string;
+  };
+  profileHref: string;
+  libraryHref: string;
+};
+
+const ResearchOutputPage: React.FC<ResearchOutputPageProps> = ({
   title,
   text,
-}) => {
-  return (
-    <div css={[containerStyles]}>
-      <TagLabel>Proposal</TagLabel>
-      <Display>{title}</Display>
-      <RichText toc text={text} />
+  team,
+  publishDate,
+  created,
+  profileHref,
+  libraryHref,
+}) => (
+  <div css={containerStyles}>
+    <div css={backContainerStyles}>
+      <Link href={libraryHref}>
+        <div css={backButtonStyles}>
+          <span css={iconStyles}>{chevronCircleLeftIcon}</span> Back
+        </div>
+      </Link>
     </div>
-  );
-};
+    <Card padding={false}>
+      <div css={cardPaddingStyles}>
+        <TagLabel>Proposal</TagLabel>
+        <Headline3 styleAsHeading={1}>{title}</Headline3>
```

Comment:
Why was this changed, `h1` seems correct in the level hierarchy to me?

Reply:
James requested it on the trello card

Reply:
Then you probably meant to write `Display styleAsHeading={3}`

Reply:
Derp.